google_cse.module

```diff
@@ -230,8 +233,21 @@ function google_cse_form_search_block_form_alter(&$form, &$form_state, $form_id)
   /** @var \Drupal\search\SearchPageRepositoryInterface $repo */
   $repo = \Drupal::service('search.search_page_repository');
   if ($repo->getDefaultSearchPage() == 'google_cse_search_type') {
+    $config = \Drupal::config('google_cse.settings'); 
     google_cse_sitesearch_form($form);
     $form['#attributes']['class'][] = 'google-cse';
```

Comment:
that duplicates code in hunk above, suppose better move this attribute assignment inside `google_cse_sitesearch_form()`